packages/open-next/src/converters/edge.ts

```diff
@@ -68,7 +72,13 @@ const converter: Converter<
         },
       });
 
-      return fetch(req);
+      const cfCache = result.isISR ? { cacheEverything: true } : {};
+
+      return fetch(req, {
+        // This is a hack to make sure that the response is cached by Cloudflare
+        // @ts-expect-error - This is a Cloudflare specific option
```

Comment:
suggestion: reference the CF docs: `https://developers.cloudflare.com/workers/examples/cache-using-fetch/#caching-html-resources`